wporg/wporg.go

```diff
@@ -10,6 +10,8 @@ import (
 	"errors"
 )
 
+const TimeFormat = "2006-01-02 3:04pm MST"
```

Comment:
Why is this an actual date and not a format pattern?

Reply:
That is Go's format pattern. Seriously. Its weird.

https://golang.org/pkg/time/#Time.Format